pkg/backend/backenderr/backenderr.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/util/env"
 )
 
+var ErrNotFound = NotFoundError{}
```

Comment:
This doesn't seem to be used anywhere.

Reply:
It's used in `pkg/backend/httpstate/package_registry.go`.

Reply:
No, it isn't?  I see `NotFoundError` used there in https://github.com/pulumi/pulumi/pull/19529/files#diff-1890810a3497319171c687c82623583ab335c2e35635445f8eb14814b62cecb9R56, but not `ErrNotFound`?

Reply:
Ah, right. The general custom that I'm aware of in Go when introducing a new error is to expose the error type as `*Error` and expose a zero value instance of that type as `Err*`. The latter is to allow `errors.Is(err, backenderr.ErrNotFound)`.

It isn't used here, but will be used in follow up PRs.